src/PoshGitTypes.ps1

```diff
@@ -244,8 +244,9 @@ class GitPromptSettings {
     [Nullable[bool]]$EnableFileStatusFromCache        = $null
     [string[]]$RepositoriesInWhichToDisableFileStatus = @()
 
-    [string]$DescribeStyle       = ''
-    [psobject]$EnableWindowTitle = 'posh~git ~ '
+    [string]$DescribeStyle   = ''
+    [bool]$EnableWindowTitle = $true
```

Comment:
> Also, I think we might want to keep `EnableWindowTitle` as a simple bool. If you disable the WindowTitle by setting it to $null, you lose the original definition.

I'm not too worried about this, since resetting is as simple as opening a new shell. Fewer settings seems better than many.

Reply:
Well, we have reduced the number of "top-level" settings a lot.  :-)  But I don't feel that strongly about this.